### PR TITLE
Document some default parameters

### PR DESCRIPTION
**Pull request by corinwagen**
Cf. #19 -- took a first pass at just writing down the most useful parameters and what their values are. Not sure if this is the best place for the file; it seemed like too much for the root ``README.md``.

Criticism/clarification is welcomed!

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# docs
```
[the remark this answers is not in the document]
Comment (kjelljorner) [reply docs/README.md:1]: I would also list:
- `imultistate` (another one of the specialities of DL-FIND)

**Review by kjelljorner — COMMENTED** [review]
Overall looks good but some minor things to correct/add. Seems like they changed the GitHub windows runners so I need to update the wheel building, but you can go ahead to make the revisions in the meantime.

File: docs/README.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+e.g., multistate optimizations sometimes change defaults relative to their single-state congeners. 
+
+#### ``printl``
+Printout level.
+- 0: none 
+- 2: some (default)
+- 4: verbose
+- 6: debug
+
+#### ``icoord``
```
[the remark this answers is not in the document]
Comment (kjelljorner) [reply docs/README.md:14]: Also, please create a link from the top README

**Comment by corinwagen** [comment]
I think I've addressed the comments. I also added documentation for ``imicroiteration``, since that's the only one from your original list that wasn't there.

**Comment by kjelljorner** [comment]
Fantastic, now I just have to butt my head against the GitHub Actions wall for a while...

**Comment by corinwagen** [comment]
Have fun :/

**Pull request merged**